docs/src/references/agent/operation-workflow.md

```diff
@@ -370,8 +369,8 @@ For example, a configuration update can be triggered with:
 ```toml
 [trigger_config_update]
 command = "config_update"
-cmd_arg.tedgeUrl = "http://127.0.0.1:8000/tedge/file-transfer/example/config_update/mosquitto-1234"
-cmd_arg.type = "mosquitto"
+input.tedgeUrl = "http://127.0.0.1:8000/tedge/file-transfer/example/config_update/mosquitto-1234"
+input.type = "mosquitto"
```

Comment:
I changed my mind regarding the name of the properties used to push/gather data to/from a sub-command.

- I prefer `input` rather than `cmd_arg` and `output` rather than `payload`.
- An alternative is to use `cmd_input` and `cmd_output`.

My motivation is two folds.
- First, using `payload` was confusing even if technically this is the payload of the invoking command that is updated. Using `output` makes clear that the focus is on extracting data from the sub-command output.
- Second, if we decide to [merge the `command` and `await-command-completion`](https://github.com/thin-edge/thin-edge.io/pull/2792#discussion_r1530410208) in a single step definition, then the two aspects (pushing data to the sub-command init state and extracting data from its final state) will be grouped together. `input` and `output` sounds then better compared to `cmd_arg` and `payload`.